project-base/storefront/components/Pages/CategoryDetail/CategoryDetailContent.tsx

```diff
@@ -15,7 +15,6 @@ import { useTypedTranslationFunction } from 'hooks/typescript/useTypedTranslatio
 import { useRouter } from 'next/router';
 import { useCallback, useRef, useState } from 'react';
 import { twJoin } from 'tailwind-merge';
-
```

Comment:
Just a huge nitpick but let's put the space here back.

Reply:
Done